src/ua/samoplavskaya/hw_arrayList/ArrayList.java

```diff
@@ -0,0 +1,80 @@
+package ua.samoplavskaya.hw_arrayList;
+
+import java.util.Arrays;
+
+public class ArrayList<T> {
+    private Object[] array;
+    private int size;
+
+    ArrayList(int size) {
+        this.size = size;
+        array = new Object[size];
+    }
+
+    //Add element to the end of array {1,2,3} -> {1,2,3,4}
+    void add(T element) {
```

Comment:
Add method doesn't add element to the beginning of the array if the beginning value of the size variable is bigger than 0